src/query/storage/promremote/options.go

```diff
@@ -44,19 +45,24 @@ func NewOptions(
 	}
 	endpoints := make([]EndpointOptions, 0, len(cfg.Endpoints))
 
-	for i, endpoint := range cfg.Endpoints {
+	for _, endpoint := range cfg.Endpoints {
 		endpointOptions := EndpointOptions{
 			name:    endpoint.Name,
 			address: endpoint.Address,
 		}
+		attr := storagemetadata.Attributes{
+			MetricsType: storagemetadata.UnaggregatedMetricsType,
+		}
 		if endpoint.StoragePolicy != nil {
-			endpointOptions.resolution = endpoint.StoragePolicy.Resolution
-			endpointOptions.retention = endpoint.StoragePolicy.Retention
+			attr.MetricsType = storagemetadata.AggregatedMetricsType
+			attr.Resolution = endpoint.StoragePolicy.Resolution
+			attr.Retention = endpoint.StoragePolicy.Retention
 			if endpoint.StoragePolicy.Downsample != nil {
 				endpointOptions.downsampleAll = endpoint.StoragePolicy.Downsample.All
 			}
 		}
-		endpoints[i] = endpointOptions
+		endpointOptions.attributes = attr
```

Comment:
Creating the entire `endpointOptions` here when you already have `attr` would look better.